front/src/applications/stdcm/utils/checkStdcmConfigErrors.ts

```diff
@@ -1,26 +1,21 @@
 import type { TFunction } from 'i18next';
 
-import type { PathStep } from 'reducers/osrdconf/types';
-import { extractHHMM } from 'utils/date';
+import type { StdcmPathStep } from 'reducers/osrdconf/types';
+import { dateToHHMMSS } from 'utils/date';
 
 import { StdcmConfigErrorTypes, ArrivalTimeTypes, type StdcmConfigErrors } from '../types';
 
 const checkStdcmConfigErrors = (
   pathfindingStateError: boolean,
-  origin: PathStep | null,
-  destination: PathStep | null,
-  pathSteps: (PathStep | null)[],
+  origin: Extract<StdcmPathStep, { isVia: false }>,
+  destination: Extract<StdcmPathStep, { isVia: false }>,
   t: TFunction
 ): StdcmConfigErrors | undefined => {
   const isOneOpPointMissing = !origin || !destination;
   if (isOneOpPointMissing) {
     return undefined;
   }
 
-  if (pathSteps.some((step) => step === null || ('uic' in step && step.uic === -1))) {
-    return { errorType: StdcmConfigErrorTypes.MISSING_LOCATION };
-  }
-
```

Comment:
We don't have any warning box anymore if I add a via and launch the simulation without filling its inputs

Reply:
Nice catch !